Tests/Microsoft.AppCenter.Test.Windows/Channel/ChannelTest.cs

```diff
@@ -711,28 +711,6 @@ public async Task HandleStorageExceptionWhenClearLogs()
             }
         }
 
-        [TestMethod]
-        public async Task CheckPendingLogsOnStart()
```

Comment:
why this test was removed?

Reply:
Because `checkPendingLogs` line where removed from Channel's constructor.